src/universal/components/TaskEditor/TaskEditor.js

```diff
@@ -168,41 +190,11 @@ class TaskEditor extends Component {
           ref={setEditorRef}
         />
         {renderModal && renderModal()}
-      </div>
+      </RootEditor>
     )
   }
 }
 
-const styleThunk = () => ({
```

Comment:
the async styles were causing the grid measurements to fail so i had to remove aphrodite from all of these